code/game/turfs/turf.dm

```diff
@@ -318,6 +318,30 @@ GLOBAL_LIST_EMPTY(station_turfs)
 
 	return FALSE
 
+/turf/attackby_secondary(obj/item/weapon, mob/user, params)
+	if(istype(weapon, /obj/item/stack/sheet/iron))
```

Comment:
Not comfortable with this, why isn't this on the iron sheets instead? Should be kept within that surface area.

Reply:
Moved the behaviour to the sheets instead